scico/flax/examples/examples.py

```diff
@@ -682,7 +682,7 @@ def runtime_error_array(type: str, idstring: str, maxdiff: float):
            entries.
     """
     raise RuntimeError(
-        f"Requested value of parameter --{type}-- does not match value "
+        f"Requested value of parameter '{type}' does not match value "
```

Comment:
For consistency with the other edits, shouldn't this be: `f"Requested value of argument '{type}'"` ...?